pack/Makefile

```diff
@@ -14,6 +14,8 @@ else ifneq (,$(wildcard /etc/debian_version))
 include $(PACKDIR)/deb.mk
 else ifneq (,$(wildcard /etc/alpine-release))
 include $(PACKDIR)/apk.mk
+else ifneq (,$(wildcard /etc/os-release))
+include $(PACKDIR)/zypper.mk
```

Comment:
Are something blocks integration of SuSE build tools right into rpm.mk like it is done for centos/fedora?

Reply:
Right, the files merged.